### DISCUSSION TITLE
Cambio indirizzo azienda emittente

### DISCUSSION THREAD

**Discussion by saxxi** [fattureincloud/api#350] [Q&A] [answered]
Cambiando l'indirizzo dell'azienda emittente le fatture dal sito notiamo che:
- Le fatture `create dal sito hanno l'indirizzo corretto`
- Le fatture `inviate da API hanno l'indirizzo precedente`

Sapete il motivo? Ringrazio anticipatamente

Nota: noi non inviamo alcuna informazione riguardante il professionista che produce la fattura.

<img width="329" alt="Screenshot 2023-09-09 at 11 03 45" src="https://github.com/fattureincloud/api/assets/504684/21c8ca0e-f3e3-49a2-8775-3bd16badd858">
<img width="971" alt="Screenshot 2023-09-09 at 11 08 55" src="https://github.com/fattureincloud/api/assets/504684/18a87a04-771d-4801-a095-951581bae739">

---

JSON Payload (parziale

```JSON
{
  "id": "...",
  "sent": "...",
  "datesend": "...",
  "clientid": "...",
  "deleted_customer_name": null,
  "number": "...",
  "prefix": "...",
  "number_format": "...",
  "datecreated": "...",
  "date": "...",
  "duedate": "...",
  "currency": "...",
  "subtotal": "...",
  "total_tax": "...",
  "total": 4060,
  "adjustment": "...",
  "addedfrom": "...",
  "hash": "...",
  "status": "...",
  "clientnote": "...",
  "adminnote": "...",
  "last_overdue_reminder": null,
  "cancel_overdue_reminders": "...",
  "allowed_payment_modes": "...",
  "token": null,
  "discount_percent": "...",
  "discount_total": "...",
  "discount_type": "...",
  "recurring": "...",
  "recurring_type": null,
  "custom_recurring": "...",
  "cycles": "...",
  "total_cycles": "...",
  "is_recurring_from": null,
  "last_recurring_date": null,
  ...
  "sale_agent": "1",
  "billing_street": "Via Milano ...",
  "billing_city": "Bologna",
  "billing_state": "BO",
  "billing_zip": "",
  "billing_country": "109",
  "shipping_street": "",
  "shipping_city": null,
  "shipping_state": null,
  "shipping_zip": null,
  "shipping_country": null,
  "include_shipping": "0",
  "show_shipping_on_invoice": "1",
  "show_quantity_as": "1",
  "project_id": "309",
  "subscription_id": "0",
  "symbol": "€",
  "name": "EUR",
  "decimal_separator": ".",
  "thousand_separator": ",",
  "placement": "before",
  "isdefault": "1",
  "currencyid": "1",
  "currency_name": "EUR",
  "custom_expenses": "1.92",
  "revenue_stamp_expenses": 2,
  "before_no_tax_on_revenue_stamp": false,
  "total_left_to_pay": "4060.00",
  "items": [...],
  "attachments": [],
  "project_data": {
    "id": "309",
    "name": "LB0150...",
    ...
    "settings": {
      ...
    },
    "client_data": {
      "address": "Via Milano ...",
      ...
    }
  },
  "visible_attachments_to_customer_found": false,
  "client": {
    ...
    "address": "Via Milano ...",
    ...
  },
  "payments": [],
  "scheduled_email": null
}
```

**Comment by emajo** [comment]
Ciao @saxxi, il json che ha inviato non è un json per le nostre api, i campi sono infatti completamente diversi da quelli accettati da noi, le consiglierei di scrivere al nostro supporto clienti all'interno di Fatture in Cloud per andare a fare un controllo più approfondito sulla problematica riscontrata.

Grazie,
Emanuele.

**Reply by valmoz** [reply, marked as answer]
Buongiorno @saxxi 
è riuscito a risolvere il problema?

Le invio la seguente guida, che sembrerebbe fare al caso vostro: 
[Problema: ho modificato i dati in impostazioni ma nei documenti vedo i dati vecchi](https://secure.fattureincloud.it/help/articolo/162-problema-modificato-dati-impostazioni-ma-documenti-vedo-dati-vecchi)

Grazie
Mauro
